src/bentoml/_internal/frameworks/catboost.py

```diff
@@ -205,7 +205,7 @@ def save_model(
         training_class_name=model.__class__.__name__,
     )
 
-    with bentoml.models.create(
+    with bentoml.models._create(  # pyright: ignore [reportPrivateUsage]
```

Comment:
/s/pyright: ignore [reportPrivateUsage]/type: ignore

So that both mypy and pyright understand this.

Reply:
Unfortunately, VS code, the IDE I'm using, does not use mypy so I have no way of testing if the ignore suppresses the warning. I also tried copying your suggestion above but it did not seem to work. 

What I can do is `# type: ignore` to ignore the line entirely instead of specifically targeting private method use. Lmk if you would like that instead.